manager/controlapi/network.go

```diff
@@ -18,34 +18,45 @@ func validateIPAMConfiguration(ipamConf *api.IPAMConfig) error {
 		return grpc.Errorf(codes.InvalidArgument, "ipam configuration: cannot be empty")
 	}
 
+	var me MultiError
+
 	_, subnet, err := net.ParseCIDR(ipamConf.Subnet)
 	if err != nil {
-		return grpc.Errorf(codes.InvalidArgument, "ipam configuration: invalid subnet %s", ipamConf.Subnet)
+		err = grpc.Errorf(codes.InvalidArgument, "ipam configuration: invalid subnet %s", ipamConf.Subnet)
```

Comment:
In all these cases, I'd suggest moving the `grpc.Errorf` wrapping from these validate functions to the place where the RPC handler returns the validation errors. That way `codes.InvalidArgument` is still visible to the client.